bindings/jupyter-modules/jupyter-ma-causal/src/containers/index.js

```diff
@@ -54,8 +55,28 @@ class App extends Component {
                 </div>
               </div>
             ))}
+
             <LineIndicatorContainer index={i} />
           </div>
+          <div style={{position: 'relative', width: columnWidth}}>
+            {' '}
+            {(data[i].lines || []).map(d => (
+              <React.Fragment key={d.name}>
+                {['treatment', 'control'].map(groupName => (
+                  <div key={groupName}>
+                    <div>{`${d.name} ${groupName} group`}</div>
+                    <div>
+                      <StaticbarContainer
+                        index={i}
+                        lineName={d.name}
```

Comment:
shall we revisit for this prop name, or at least add inlines comments on where getLineDataFactory is used? `lineName` is not indicative for what it is.

Reply:
Added comments at where getLineDataFactory is used.